### DISCUSSION TITLE
Can't access 60-day trial for GitHub Copilot

### DISCUSSION THREAD

**Discussion by drandreasdr** [community/community#54367] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I'm trying to activate my 60-day trial for GitHub Copilot, but it seems to revert back to the 30-day option. Screenshots attached.

The first screen seems to promise that if I sign up for the yearly payment, that should unlock a 60-day trial. But on the next screen, it still seems like it is still limited to 30 days.

Am I misunderstanding something here?

Best Regards,
Andreas

![image](https://user-images.githubusercontent.com/6980975/235952706-d3037046-6ea2-4a88-8bb8-7222a41c2cd8.png)
![image](https://user-images.githubusercontent.com/6980975/235952896-30f1425f-f380-47e6-9809-5fb2c66a38b4.png)

**Comment by Akash1134** [comment, marked as answer]
Hi @drandreasdr ,

Copilot is offered with a 30 days trial period as mentioned in the [docs](https://docs.github.com/en/billing/managing-billing-for-github-copilot/about-billing-for-github-copilot#about-billing-for-github-copilot). For more details, see our [FAQs](https://github.com/features/copilot#:~:text=Frequently%20asked%C2%A0questions) 

Regarding the incorrect 60 days trial frequency (I am unable to reproduce it), I've shared this with the concerned team internally. 

Thanks!
